src/meta/src/barrier/mod.rs

```diff
@@ -400,6 +407,9 @@ where
             CommandChanges::DropTables(table_ids) => {
                 assert!(self.dropping_tables.is_superset(&table_ids));
                 self.dropping_tables.retain(|a| !table_ids.contains(a));
+                // We should also clean the record in creating tables, because this changes could be
+                // issued by cancel command.
+                self.creating_tables.retain(|a| !table_ids.contains(a));
```

Comment:
IIUC, this happens if we cancel the job by pushing a `Cancel` command. At this time, the previous `Create` should already be `cancel`ed and the changes were removed. So `Cancel` will acts like a normal `Drop` here, then why do we need this change? 🤔